mono/mini/exceptions-arm64.c

```diff
@@ -123,7 +123,7 @@ mono_arch_get_call_filter (MonoTrampInfo **info, gboolean aot)
 	labels [0] = code;
 	arm_cbzx (code, ARMREG_IP0, 0);
 	for (i = 0; i < num_fregs; ++i)
-		arm_ldrfpx (code, ARMREG_D8 + i, ARMREG_R0, MONO_STRUCT_OFFSET (MonoContext, fregs) + (i * 8));
+		arm_ldrfpx (code, ARMREG_D8 + i, ARMREG_R0, MONO_STRUCT_OFFSET (MonoContext, fregs) + ((i + 8) * 8));
```

Comment:
So this starts halfway through `fregs` now? Why is `num_fregs = 8` when `fregs` is declared as `double fregs [16]`?